easybuild/easyconfigs/s/SIERRA/SIERRA-1.1.12-foss-2021b.eb

```diff
@@ -0,0 +1,98 @@
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('matplotlib', '3.4.3'),
+    ('PyYAML', '5.4.1'),
+    ('sympy', '1.9'),
+    ('ARGoS', '3.0.0-beta59'),
+    ('parallel', '20210722'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_filter = ("python -c 'import %(ext_name)s''", '')
+exts_list = [
+    ('Sphinx', '4.5.0', {
+        'checksums': ['7bf8ca9637a4ee15af412d1a1d9689fec70523a68ca9bb9127c2f3eeb344e2e6'],
+    }),
+    ('similaritymeasures', '0.4.4', {
+        'checksums': ['752859e051b3656171505bd0413b6ca89601ef267d9ec385117e399b1c81c8f2'],
+    }),
+    ('fastdtw', '0.3.4', {
+        'checksums': ['2350fa6ec36bcad186eaf81f46eff35181baf04e324f522de8aeb43d0243f64f'],
+    }),
+    ('humanfriendly', '10.0', {
+        'checksums': ['6b0b831ce8f15f7300721aa49829fc4e83921a9a301cc7f606be6686a2288ddc'],
+    }),
+    ('coloredlogs', '15.0.1', {
+        'checksums': ['7c991aa71a4577af2f82600d8f8f3a89f936baeaf9b50a9c197da014e5bf16b0'],
+    }),
+    ('singleton-decorator', '1.0.0', {
+        'checksums': ['1a90ad8a8a738be591c9c167fdd677c5d4a43d1bc6b1c128227be1c5e03bee07'],
+    }),
+    ('implements', '0.3.0', {
+        'checksums': ['1e93cf9407f8af127724717376b8e16c2e8b5ce8afbe6b1f4bf113d02e124835'],
+    }),
+    ('retry', '0.9.2', {
+        'checksums': ['f8bfa8b99b69c4506d6f5bd3b0aabf77f98cdb17f3c9fc3f5ca820033336fba4'],
+    }),
+    ('distro', '1.7.0', {
+        'checksums': ['151aeccf60c216402932b52e40ee477a939f8d58898927378a02abbe852c1c39'],
+    }),
+    ('sphinx-argparse', '0.3.1', {
+        'modulename': False,
+        'checksums': ['82151cbd43ccec94a1530155f4ad34f251aaca6a0ffd5516d7fadf952d32dc1e'],
+    }),
+    ('sphinx_rtd_theme', '1.0.0', {
+        'checksums': ['eec6d497e4c2195fa0e8b2016b337532b8a699a68bcb22a512870e16925c6a5c'],
+    }),
+    ('pockets', '0.9.1', {
+        'checksums': ['9320f1a3c6f7a9133fe3b571f283bcf3353cd70249025ae8d618e40e9f7e92b3'],
+    }),
+    ('sphinxcontrib-napoleon', '0.7', {
+        'modulename': False,
+        'checksums': ['407382beed396e9f2d7f3043fad6afda95719204a1e1a231ac865f40abcbfcf8'],
+    }),
+    ('autoapi', '2.0.1', {
+        'checksums': ['4003b17599020652d0738dc1c426d0abf2f58f8a1821f5500816043210b3d1d6'],
+    }),
+    ('sphinx-tabs', '3.3.1', {
+        'checksums': ['d10dd7fb2700329b8e5948ab9f8e3ef54fff30f79d2e42cfd1b0089ae26e8c5e'],
+    }),
+    ('sphinx-last-updated-by-git', '0.3.2', {
+        'checksums': ['71300227d322cab4733fc1da4e34f50d51b244c45e44af5adfc2fe219112c88d'],
+    }),
+    (name, version, {
+        'preinstallopts': "cd docs && make man && cd .. && ",
+        'source_tmpl': '%s.tar.gz' % _commit,
+        'source_urls': ['https://github.com/swarm-robotics/sierra/archive'],
+        'checksums': ['71dc8a62f84191fceed9e90aea60860038f4ba7ed63f965aad8f1b4028533f73'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/sierra-cli'],
+    'dirs': [],
+}
+
```

Comment:
Is there a sensible `sanity_check_commands`  you can define? (e.g. a `--version`)

Reply:
no, as far as I understand you need an "experiment" to have been setup for the commands to output anything useful

Reply:
Ok